packages/playground/blueprints/src/lib/steps/set-site-language.ts

```diff
@@ -19,6 +22,40 @@ export interface SetSiteLanguageStep {
 	language: string;
 }
 
+/**
+ * Returns the URL to download a WordPress translation package.
+ *
+ * If the WordPress version doesn't have a translation package,
+ * the latest "RC" version will be used instead.
+ */
+const getWordPressTranslationUrl = (wpVersion: string, language: string) => {
+	/**
+	 * The translation API provides translations for all WordPress releases
+	 * including patch releases.
+	 *
+	 * RC and beta versions don't have individual translation packages.
+	 * They all share the same "RC" translation package.
+	 *
+	 * Nightly versions don't have a "nightly" translation package.
+	 * So, the best we can do is download the RC translation package,
+	 * because it contains the latest available translations.
+	 *
+	 * The WordPress.org translation API uses "RC" instead of
+	 * "RC1", "RC2", "BETA1", "BETA2", etc.
+	 *
+	 * For example translations for WordPress 6.6-BETA1 or 6.6-RC1 are found under
+	 * https://downloads.wordpress.org/translation/core/6.6-RC/en_GB.zip
+	 */
+	const wpVersionString = versionStringToLoadedWordPressVersion(wpVersion);
+	if (wpVersionString === 'nightly' || wpVersionString === 'beta') {
+		wpVersion = MinifiedWordPressVersions['beta'].replace(
```

Comment:
A dedicated version string conversion logic might make more sense than post-processing the result of another function that's implemented for a different purpose.